test/spinxsk/spinxsk.c

```diff
@@ -1992,6 +2025,308 @@ ProcessPkts(
     return TRUE;
 }
 
+//
+// Add some "corrupted" bytes to a buffer.
+// Each byte of the buffer has a 1/20 chance to be replaced by a random value.
+//
+VOID
+CorruptBuffer(
```

Comment:
I would call this "PartiallyRandomizeBuffer" since "corrupt" is somewhat ambiguous in networking and might imply messing with the underlying buffer allocation.